packages/wonder-blocks-dropdown/src/components/__tests__/multi-select.test.tsx

```diff
@@ -1683,12 +1683,7 @@ describe("MultiSelect", () => {
             };
 
             const {userEvent} = doRender(
-                <MultiSelect
-                    onChange={jest.fn()}
-                    isFilterable={true}
```

Comment:
Does including `isFilterable` result in a different behaviour? Would a test covering both cases be helpful?

Reply:
It should have the same behavior for this scenario! There isn't any logic for isFilterable around announcing the number of options when the listbox is open.